x-pack/plugins/reporting/public/lib/stream_handler.ts

```diff
@@ -8,7 +8,8 @@ import { i18n } from '@kbn/i18n';
 import * as Rx from 'rxjs';
 import { catchError, map } from 'rxjs/operators';
 import { NotificationsSetup } from 'src/core/public';
-import { JobId, JobStatusBuckets, JobSummary, SourceJob } from '../../common/types';
+import { JobStatusBuckets, JobStatusBucket } from '../';
```

Comment:
I'm not sure what I think of `JobStatusBucket`, buckets are a hard one to stomach for those unfamiliar. WDTY?

Reply:
I'll go with `ReportStatus` / `ReportStatusSet`